fh_immuta_utils/policy.py

```diff
@@ -144,6 +144,7 @@ class SubscriptionPolicyAction(BaseModel):
     subscriptionType: str
     exceptions: Optional[PolicyExceptions] = None
     allowDiscovery: bool = False
+    automaticSubscription: bool = True
```

Comment:
For backward compatibility, should default be False?

Reply:
The default in Immuta is True. Tested by enrolling a policy without this parameter specified in the configs. You can also see it in the UI when creating a new subscription policy.